tests/cpp/test_cmake.py

```diff
@@ -0,0 +1,78 @@
+#
+#   Copyright 2013-2023 The Foundry Visionmongers Ltd
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+"""
+Test CMake installed package.
+Python script invoked from Ctest, in order to test structure
+and contents of cmake installed package. Particularly with a focus
+on the .dist_info metadata required to make this package play well
+with pip.
+"""
+# pylint: disable=missing-function-docstring
+import os
+
+import pytest
+
+import sys
+
+
+try:
+    from importlib import metadata
+except:
+    import importlib_metadata as metadata
```

Comment:
Removed with my trash debugging statments : https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/66/commits/2421f8e980aee79b83aae6a288d08552b6728326

Reply:
Oh right, i thought it was a library.

I did take this directly from https://github.com/feltech/OpenAssetIO/blob/765ee0c6d5183de59dd198318633f7c124ad20ed/src/openassetio-python/tests/test_cmake.py, this'll make it inconsistent. I suspect the same logic about conditional imports was done in the other file and it was set that way for a reason.

Reply:
OK fair enough. I don't recall if there was a good reason for locating the conditional import there, but like I say, it's a grey area since we have logic mixed in with imports. Let's leave as-is, thanks.